Formula/freeling.rb

```diff
@@ -29,7 +29,6 @@ def install
       end
 
       bootstrap_args = %W[
-        --without-icu
```

Comment:
There is a `--with-icu=` 3 lines below which will take precedence over this redundant one.